include/votca/xtp/cudamatrix.h

```diff
@@ -48,13 +48,34 @@ cudaError_t checkCuda(cudaError_t result);
 
 Index count_available_gpus();
 
+template <class M>
```

Comment:
`CudaMatrixTranspose` only differs of `CudaMatrix` in the flag that identifies whether or not the matrix is transpose? I am sure that I am missing something behind the reason to create this new clas.

Reply:
Yes, exactly. I wanted to reproduce the `Eigen` synatx without actually modifying the object, so calling `transpose()` creates a wrapper object, which shifts the transpose flag.